### PR TITLE
Fix errors created by iTwin 3.1.

### PR DESCRIPTION
**Pull request by tcobbs-bentley**

### CODE REVIEW

File: cross-platform/react-app/package.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
   "private": true,
   "homepage": "./",
   "scripts": {
-    "start": "USE_FAST_SASS=true BROWSER=none react-scripts start",
+    "start": "GENERATE_SOURCEMAP=false USE_FAST_SASS=true BROWSER=none react-scripts start",
```
Comment (aruniverse) [inline cross-platform/react-app/package.json:12]: you migth also want to set these two for start & build

```
TRANSPILE_DEPS=false
DISABLE_TERSER=true
```

Comment (tcobbs-bentley) [reply cross-platform/react-app/package.json:12]: Can you explain what those do?

Comment (aruniverse) [reply cross-platform/react-app/package.json]: https://github.com/imodeljs/create-react-app/blob/imodeljs/packages/react-scripts/README-imodeljs.md

Comment (tcobbs-bentley) [reply cross-platform/react-app/package.json]: Updated.

**Review by toddsouthenbentley — APPROVED** [review]

**Pull request merged**